src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -163,6 +171,10 @@ const SpeciesMainViewStats = (props: Props) => {
     }
   }, [props.genomeStats, props.activeGenomeId, props.exampleFocusObjects]);
 
+  const expandedSections = props.genomeUIState
+    ? [...props.genomeUIState.expandedSections]
```

Comment:
Is there any reason to clone the array here? I see that you are trying to mutate this array in the `onSectionToggle` — perhaps the cloning should be done there?

```
props.setExpandedSections([...expandedSections, section]);
```

Reply:
That's done